app.py

```diff
@@ -255,24 +258,24 @@ def get(self):
             filename {str} -- The filename to upload. Should be the tmpfile
                               created by `write_data`
             tracking_id {str} -- The tracking ID sent by the client
-            hash_value {str} -- the uuid for this upload generated by us at time of POST
+            payload_id {str} -- the uuid for this upload generated by us at time of POST
```

Comment:
It’s not generated by us, but by 3Scale. Moreover, is it really a UUID now, or did it become just an arbitrary unique ID?